lib/views/home/widgets/menu/row_item_widget.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:Medito/constants/constants.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+
+class RowItemWidget extends StatelessWidget {
+  const RowItemWidget({
+    super.key,
+    required this.title,
+    required this.iconCodePoint,
+    this.isShowUnderline = true,
```

Comment:
Yeah this should be "shouldShowUnderline". "isShowUnderline" is confusing, 
"isUnderlined" or "hasUnderline" also work

Reply:
hmm makes sense

Reply:
Changed to "hasUnderline"